ballsdex/packages/players/cog.py

```diff
@@ -29,43 +31,55 @@ def __init__(self, bot: "BallsDexBot"):
                 0
             ]._Parameter__parent.choices.pop()  # type: ignore
 
+    friend = app_commands.Group(name="friend", description="Friend commands")
+    blocked = app_commands.Group(name="block", description="Block commands")
+    donation = app_commands.Group(name="donation", description="Donation commands")
+    mention = app_commands.Group(name="mention", description="Mention commands")
+
     @app_commands.command()
     @app_commands.choices(
         policy=[
             app_commands.Choice(name="Open Inventory", value=PrivacyPolicy.ALLOW),
             app_commands.Choice(name="Private Inventory", value=PrivacyPolicy.DENY),
             app_commands.Choice(name="Same Server", value=PrivacyPolicy.SAME_SERVER),
+            app_commands.Choice(name="Friends Only", value=PrivacyPolicy.FRIENDS),
```

Comment:
```suggestion
            app_commands.Choice(name="Friends Only", value=PrivacyPolicy.FRIENDS),
            app_commands.Choice(name="Same Server", value=PrivacyPolicy.SAME_SERVER),
```

if members intent is disabled, we pop the final option from this set of choices (which before was same server) so when a new option was added at the end it was getting rid of that instead of same server. Changing the order fixes this